doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,471 @@
+
+The code for integrating MCUboot into |NCS| is located in :file:`ncs/nrf/modules/mcuboot`.
+
+The following list summarizes the most important changes inherited from upstream MCUboot:
+
+* Added support for indicating serial recovery through LED.
+* Made the debounce delay of the serial detect pin state configurable.
+* Added support for mbed TLS ECDSA for signatures.
+* Added an option to use GPIO PIN to enter to USB DFU class recovery.
+* Added an optional check that prevents attempting to boot an image built for a different ROM address than the slot it currently resides in.
+  The check is enabled if the image was signed with the ``IMAGE_F_ROM_FIXED`` flag.
+
+nrfx
+----
+
+See the `Changelog for nrfx 2.5.0`_ for detailed information.
+
+Mcumgr
+======
+
+The mcumgr library contains all commits from the upstream mcumgr repository up to and including snapshot ``74e77ad08``.
+
+The following list summarizes the most important changes inherited from upstream mcumgr:
+
+* No changes yet
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``730acbd6ed`` (``v2.6.0-rc1``), plus some |NCS| specific additions.
+
+For a complete list of upstream Zephyr commits incorporated into |NCS| since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline v2.6.0-rc1 ^v2.4.99-ncs1
+
+For a complete list of |NCS| specific commits, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^v2.6.0-rc1
+
+The current |NCS| release is based on Zephyr v2.6.0-rc1.
+See the :ref:`zephyr:zephyr_2.6` Release Notes for an overview of the most important changes inherited from upstream Zephyr.
+
+Matter (Project CHIP)
+=====================
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to and including ``aa96ea0365``.
+
+The following list summarizes the most important changes inherited from the upstream Matter:
+
+* Added:
+
+  * Completed the persistent storage feature, which allows Matter devices to successfully communicate with each other even after reboot.
+  * Added support for OpenThread's Service Registration Protocol (SRP) to enable the discovery of Matter nodes using the DNS-SD protocol.
+  * Added support for Network Commissioning Cluster, used when provisioning a Matter node.
+  * Enabled Message Reliability Protocol (MRP) for the User Datagram Protocol (UDP) traffic within a Matter network.
+  * Added support for Operational Credentials Cluster, used to equip a Matter node with an operational certificate.
+
+Documentation
+=============
+
+There are no entries for this section yet.
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v1.5.0`_ for the list of issues valid for this release.
```

Comment:
Not sure if this is a temporary thing to get the documentation building, but in case it isn't:

```suggestion
See `known issues for nRF Connect SDK v1.6.0`_ for the list of issues valid for this release.
```

Reply:
Yes, that will be fixed